source/SH3/system/glprogram.cpp

```diff
@@ -39,7 +41,7 @@ GLint sh3_glprogram::Load(const std::string& name)
 
         errorLog.resize(logLength);
         glGetProgramInfoLog(programID, logLength, &logLength, &errorLog[0]);
-        glDeleteProgram(vertShader);
+        glDeleteShader(vertShader);
         glDeleteShader(fragShader);
 
         die(&errorLog[0]);
```

Comment:
When the string contains something looking like a format specifier, things will go wrong. Use `die("%s", ...)` instead.